Simplenote/SPTextView.m

```diff
@@ -15,7 +15,6 @@
 
 #define kMaxEditorWidth 750 // Note: This matches the Electron apps max editor width
 NSString *const MarkdownUnchecked = @"- [ ]";
```

Comment:
In the interest of having a single source of truth, maybe we could nuke this const? It's only used one other place in this file. Maybe make NSAttributedStringToMarkdownConverter `@objc` friendly and its static members not private and reference `NSAttributedStringToMarkdownConverter.unchecked` @ line 140?

Reply:
+10000 for canonical source of truth!! Hoooowever (apologies I didn't mention this in the PR description), the other two spots in which is is used will be removed / heavily refactored in a follow up.

Thank you for bringing this up Eric!!